### ISSUE TITLE
How to concatenate x1 and x2 in HyperDenseNet

### ISSUE THREAD

**Issue by rabin9948** [josedolz/LiviaNET#14]
Hi.

Studying your HyperDenseNet, I am curious about how to concatenate x1( size of x1 is 27x27x27 ) and x2( size of x2 is 25x25x25 ) in HyperDenseNet. I think that to concatenate x1 and x2, we extract 25x25x25 volumes around the center of x1. Is that right? Thanks!

**Comment by josedolz** [comment]
Hi, 

yes, this is exactly what we are doing to concatenate feature maps with different sizes.

Best

**Comment by rabin9948** [comment]
Thank you for the answer! 
If I have more questions, I will write again.

**Comment by rabin9948** [comment]
I have one question. I am curious about the learning rate decay. because overfitting problem arises.  The initial learning rate to 0.001 is reduced by a factor of 2 after every 5 epochs ( starting from epoch 10 ). So, when epoch is 10, the learning rate is  0.001*0.5. And when epoch is 15, the learning rate is 0.001*0.5*0.5. Is that way right?
